packages/daimo-api/src/server/pushNotifier.ts

```diff
@@ -218,12 +217,18 @@ export class PushNotifier {
     }
     const otherStr = getAccountName(otherAcc);
 
-    const title = value < 0 ? `Sent $${dollars}` : `Received $${dollars}`;
+    const title =
+      value < 0
+        ? `Sent $${dollars} to ${otherStr}`
+        : `Received $${dollars} from ${otherStr}`;
     let body;
-    if (value < 0) {
-      body = `You sent ${dollars} ${tokenSymbol} to ${otherStr}`;
+
+    if (memo) {
+      body = memo;
     } else if (receivingRequestedMoney) {
-      body = `${otherStr} fulfilled your ${dollars} ${tokenSymbol} request`;
+      body = `Your ${dollars} ${tokenSymbol} request was fulfilled`;
+    } else if (value < 0) {
+      body = `You sent ${dollars} ${tokenSymbol} to ${otherStr}`;
```

Comment:
Maybe these should be `$${dollars}` for consistency with sent/received above ^

Reply:
discussed IRL, this body is the same as before -- the change here is to *not* show this if there's a memo and put the memo text in body instead